pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -2330,6 +2333,33 @@ class FixProcessor {
     _addFix(DartFixKind.LINT_ADD_OVERRIDE, []);
   }
 
+  void _addLintFixReplaceWithCollectionLiteral() {
+    final InstanceCreationExpression instanceCreation =
+        node.getAncestor((node) => node is InstanceCreationExpression);
+    final InterfaceType type = instanceCreation.staticType;
+    final generics = type.typeArguments;
+    final buffer = new StringBuffer();
+    var addGenerics = false;
+    for (final generic in generics) {
+      if (generic.name != 'dynamic') {
+        addGenerics = true;
+        break;
+      }
+    }
+    if (addGenerics) {
```

Comment:
It's better to copy the original source when possible because then we don't drop comments or change formatting. Better to use `utils.getNodeText(generics)`.